event-handlers/src/main/java/no/sikt/nva/nvi/events/evaluator/EvaluatorService.java

```diff
@@ -100,7 +105,13 @@ public Optional<CandidateEvaluatedMessage> evaluateCandidacy(URI publicationBuck
         }
     }
 
+    private static boolean hasCreatorsWithNviAffiliations(List<VerifiedNviCreator> verifiedCreatorsWithNviInstitutions,
+                                                          List<UnverifiedNviCreator> unverifiedCreatorsWithNviInstitutions) {
+        return !verifiedCreatorsWithNviInstitutions.isEmpty() || !unverifiedCreatorsWithNviInstitutions.isEmpty();
+    }
```

Comment:
Maybe rename method to `hasCreators()` as it does not do any checks on nvi affiliations